<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettings</class>
 <widget class="QWidget" name="AppearanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="iconSizeBox">
       <property name="title">
        <string>Icon Size</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="smallRadioButton">
            <property name="text">
             <string>small</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mediumRadioButton">
            <property name="text">
             <string>medium</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="largeRadioButton">
            <property name="text">
             <string>large</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="iconStyleBox">
       <property name="title">
        <string>Icon Style</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="justTextRadioButton">
            <property name="text">
             <string>just text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="justIconRadioButton">
            <property name="text">
             <string>just icons</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="textAndIconsRadioButton">
            <property name="text">
             <string>text and icons</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="windowStateBox">
       <property name="title">
        <string>Window State</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="windowStateCheckBox">
            <property name="text">
             <string>Save window size and position on exit.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="textEditorBox">
       <property name="title">
        <string>Text Editor</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="fontSizeTextEditorLabel">
            <property name="text">
             <string>Font Size in Text Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fontSizeTextEditorLabelSpinBox">
            <property name="minimum">
             <number>9</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="informationBoardBox">
       <property name="title">
        <string>Information Board</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="fontSizeInformationBoardLabel">
            <property name="text">
             <string>Font Size in Information Board</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fontSizeInformationBoardSpinBox">
            <property name="minimum">
             <number>9</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
